kratos/geometries/hexahedra_3d_8.h

```diff
@@ -603,6 +603,40 @@ template<class TPointType> class Hexahedra3D8 : public Geometry<TPointType>
             MathUtils<double>::Norm3(p3-p7)) /12.0;
     }
 
+    /** This method calculates and returns the maximum edge
+     * length of the geometry
+     *
+     * @return double value with the maximum edge length
+     *
+     * @see MinEdgeLength()
+     * @see AverageEdgeLength()
+     */
+    double MaxEdgeLength() const override {
+        const auto edges = GenerateEdges();
+        double max_edge_length = -std::numeric_limits<double>::max();
```

Comment:
```suggestion
        double max_edge_length = std::numeric_limits<double>::lowest();
```

Reply:
Huuuum, Now I am confused

Reply:
good point :D

Reply:
lowest might not work with an all 0.00 distance, the idea is to use something negative, and as per code coherency I used -`-std::numeric_limits<double>::max()` the biggest negative number you can get. In fact both max and minimum were already computed in the `ShortestToLongestEdgeQuality`